### PR TITLE
py3 - ported ec2_vpc_route_table iterkeys to dict.keys()

### PR DESCRIPTION
**Pull request by Shaps**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
ec2_vpc_route_table

##### ANSIBLE VERSION
N/A
##### SUMMARY
Addresses ansible/ansible#18507

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Shaps. To the current maintainers, @erydo, @wimnat please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/amazon/ec2_vpc_route_table.py [hunk 1]
```diff
@@ -299,7 +299,7 @@ def route_spec_matches_route(route_spec, route):
             if all((not route.gateway_id, not route.instance_id, not route.interface_id, not route.vpc_peering_connection_id)):
                 return True
 
-    for k in key_attr_map.iterkeys():
+    for k in key_attr_map.keys():
```
Comment (abadger) [inline cloud/amazon/ec2_vpc_route_table.py]:
``` python

this can be made even simpler as:
``` python
for k in key_attr_map:
```

**Comment by abadger** [comment]
needs_revision

**Comment by gregdek** [comment]
Thanks @Shaps for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by Shaps** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @Shaps. To the current maintainers, @erydo, @wimnat please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by abadger** [comment]
Thanks @Shaps !  Merged to devel and stable-2.2.  Will either be in 2.2.1 or 2.2.2.

**Pull request merged**